### PR TITLE
ZKUI-13 switch to github actions

### PR DESCRIPTION
**Pull request by tcarmet**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        uses: docker/build-push-action@v2.7.0
+        with:
+          context: .
+          push: true
+          tags: "registry.scality.com/zenko-ui/zenko-ui:${{ github.event.inputs.tag }}"
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v0.1.13
```
Comment (miniscruff) [inline .github/workflows/release.yaml:49]: this was updated in pensieve like an hour ago, I think it is v1 now

File: .github/dependabot.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (miniscruff) [inline .github/dependabot.yaml:1]: while you are here you might want to add the javascript package

File: .github/workflows/release.yaml (hunk 1 above)
Comment (tcarmet) [reply .github/workflows/release.yaml:49]: the maintainer of this action is tagging v1 as 0.1.13. I would argue it's rather a v0, but ok. Anyway I prefer to leave it this way.

File: .github/dependabot.yaml (hunk 2 above)
Comment (tcarmet) [reply .github/dependabot.yaml:1]: done

File: .github/workflows/release.yaml (hunk 1 above)
Comment (miniscruff) [reply .github/workflows/release.yaml:49]: You sure? I am pretty sure dependabot will open a PR next tick

**Review by miniscruff — APPROVED** [review]

**Review by nicolas2bert — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        with:
+          node-version: '10'
+          cache: 'npm'
+
+      - name: install dependencies
+        run: npm config set unsafe-perm true && npm ci
+
+      - name: build assets
+        run: npm run build
+
```
Comment (nicolas2bert) [inline .github/workflows/release.yaml:38]: are the 2 line breaks on purpose?

Comment (tcarmet) [reply .github/workflows/release.yaml:38]: no they're not, let me fix that.

**Pull request merged**